bitnami/airflow/templates/deployment-scheduler.yaml

```diff
@@ -60,8 +60,13 @@ spec:
             - -ec
             - |
               {{- if .Values.airflow.cloneDagFilesFromGit.enabled }}
+              {{- if and .Values.airflow.cloneDagFilesFromGit.repository .Values.airflow.cloneDagFilesFromGit.branch }}
                 git clone {{ .Values.airflow.cloneDagFilesFromGit.repository }} --branch {{ .Values.airflow.cloneDagFilesFromGit.branch }} /dags
               {{- end }}
+              {{- range .Values.airflow.cloneDagFilesFromGit.repositories }}
+                git clone {{ .repository }} --branch {{ .branch }} /dags-{{ .name | kebabcase }}
+              {{- end }}
+              {{- end }}
```

Comment:
Apparently we have removed `airflow.cloneDagFilesFromGit.repository` in the README, same with `.branch` and `.path`. Therefore, we should keep those variables in the README to avoid a breaking change.

Similar changes have been made above as well, so these comments would also be valid there too.